website/docs/topics/non-openai-models/cloud-based-proxy-servers.md

```diff
@@ -0,0 +1,187 @@
+# Cloud-based Proxy Servers
+When cloud-based proxy servers provide an OpenAI-compatible API, using them in AutoGen
+is straightforward. With [LLM Configuration](/docs/topics/llm_configuration) done in the same way
+as when using OpenAI's models, the primary difference is typically the authentication which is
+usually handled through an API key.
+
+An example of a cloud-based proxy server provider that has an OpenAI-compatible API is provided
+below.
+
+## Together AI
+![together.ai](images/togetherai250.png)
```

Comment:
We don't need to add their logo here for now as their logos can change overtime.